documentation/src/main/asciidoc/topics/proc_configuring_memory_size_bytes.adoc

```diff
@@ -0,0 +1,60 @@
+[id='configure_eviction_size-{context}']
+= Configuring the Maximum Amount of Memory for {brandname} Caches
+Limit the size of the data container for cache entries to a maximum amount of
```

Comment:
Should we mention here that the storage media type has to be a binary based one?